### DISCUSSION TITLE
CAPITAL repository doesn't appear in search result.

### DISCUSSION THREAD

**Discussion by cloudy-sfu** [community/community#56959] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am searching for my repository on the "overview" page of my account. The name of the targeted repository includes "OR".

I can find it by searching "or":

![Snipaste_2023-06-02_05-08-12](https://github.com/community/community/assets/41314224/2253593a-3b09-43fe-bd0c-70c0a5f93d3d)

But I cannot find it by searching "OR":

![Snipaste_2023-06-02_05-08-45](https://github.com/community/community/assets/41314224/4855f9c0-46e4-4c43-8746-377dfd28ff7a)

Why?

**Comment by look** [comment, marked as answer]
Hi @cloudy-sfu, thanks for the report. This bug is happening because `OR` is being interpreted as a search operator (boolean OR). I've filed a bug with the team. As a work around, if you quote your search term (`"OR"`) or add the next character (`OR-`) it will work.

**Comment by cloudy-sfu** [comment]
Thank you!
